portal-ui/src/screens/Console/Common/FormComponents/DateTimePickerWrapper/DateTimePickerWrapper.tsx

```diff
@@ -258,14 +262,18 @@ const DateTimePickerWrapper = ({
     };
   }
 
-  let classOverriden = "";
+  let classOverriden = `${classNamePrefix}date-time-input `;
```

Comment:
Let's avoid concatenating strings this way, it is better to declare a new variable and use ternary comparison

```suggestion
  const classOverriden = `${classNamePrefix}date-time-input  ${forSearchBock ? classes.dateSelectorOverride : ""} ${forFilterContained && !forSearchBock ? classes.dateSelectorFilterOverride : ""}`;
```